### PR TITLE
Documentation of traversal problems

### PR DESCRIPTION
**Pull request by ANUSHKA-AARYA**
In this documentation, I have explained about wave and spiral traversal problems using 2D array in c language.

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Suvraneel** [comment]
@gunjan-g

**Comment by ANUSHKA-AARYA** [comment]
@Suvraneel please review this

File: C_CPP/Data Structures/Array/TRAVERSAL PROBLEMS/wave_spiral_traversal.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+```
+## ALGORITHM
+1. First of all, enter the serial no. of all monuments .
+
+2. From fig. we can see ,for all even value of column no. we are moving downwards and for odd values of column i.e. j ,we are moving upwards .
+
+3. Apply for loop ,and write condition for odd and even according to above logic.
+
+## CODE
+```
```
Comment (gunjan-g) [inline C_CPP/Data Structures/Array/TRAVERSAL PROBLEMS/wave_spiral_traversal.md:27]:
```suggestion
```cpp
```
Use cpp markdown for codes written in cpp

**Review by gunjan-g — CHANGES_REQUESTED** [review]
@ANUSHKA-AARYA Please include proper spacing inside the codes for better view

**Comment by ANUSHKA-AARYA** [comment]
@gunjan-g  kindly review this....

**Comment by gunjan-g** [comment]
> @gunjan-g kindly review this....

Please fetch from the cpp branch first and then commit your changes

**Comment by ANUSHKA-AARYA** [comment]
@gunjan-g

**Comment by gunjan-g** [comment]
@ANUSHKA-AARYA Have you pushed changes from the same branch you've used before to contribute to some other domain?

**Comment by ANUSHKA-AARYA** [comment]
@gunjan-g  no, i have made the changes you said, and simply pushed in the current domain. You once check array folder in data structures contribution is there

**Comment by Suvraneel** [comment]
@ANUSHKA-AARYA 
These are diverged branches... The branch from where you made the PR is not based off CPP...
image

**Comment by gunjan-g** [comment]
@ANUSHKA-AARYA You can make a new pr

**Comment by Suvraneel** [comment]
That new PR must not be from this or similar hybrid branch though

**Comment by ANUSHKA-AARYA** [comment]
@gunjan-g ok

**Comment by ANUSHKA-AARYA** [comment]
@Suvraneel  sry, but  I am not getting what you are saying and i am unable to work on my new issue it is not fetching upstream.

**Comment by ANUSHKA-AARYA** [comment]
@Suvraneel

**Comment by Suvraneel** [comment]
@ANUSHKA-AARYA  If you're being unable to fetch upstream, it means you've somehow messed up & caused conflicts....
That could be solved in a session ofc...

But I have a better solution... Delete your fork, refork & then make a fresh PR

**Pull request closed without merging**